api/v1/flow/flow.proto

```diff
@@ -353,6 +353,7 @@ enum DropReason {
     NAT46 = 187;
     NAT64 = 188;
     AUTH_REQUIRED = 189;
+    NO_EGRESS_GATEWAY = 194;
```

Comment:
should we keep it sequential here?

Reply:
cc @rolinh, not really sure what's better for backports

Reply:
> should we keep it sequential here?

No we shouldn't, we need the same value as in https://github.com/cilium/cilium/pull/24835. Otherwise API consumers (e.g. Hubble CLI) could be confused as DropReason 190 would be ambiguous depending on the Cilium version,  `CT_NO_MAP_FOUND` (Cilium v1.14+)  vs `NO_EGRESS_GATEWAY` (Cilium v1.13).